raster-foundry/raster-foundry
# Manage sbtopts without Git marking changes as uncommitted

## Overview

Manage sbtopts without Git marking changes as uncommitted.

Resolves #4333 

## Testing Instructions

On your workstation, run:

```bash
$ export RF_SETTINGS_BUCKET=rasterfoundry-development-config-us-east-1
$ export AWS_PROFILE=raster-foundry
$ ./scripts/bootstrap --sbtopts
```
See that `$ git status` shows a modification that won't be committed (which reflects the state of the project):

```bash
> rbreslow@maiden raster-foundry (feature/jrb/sbtopts-mgmt) $ git status
On branch feature/jrb/sbtopts-mgmt
Your branch is up to date with 'origin/feature/jrb/sbtopts-mgmt'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)

	modified:   app-backend/.sbtopts

no changes added to commit (use "git add" and/or "git commit -a")

> rbreslow@maiden raster-foundry (feature/jrb/sbtopts-mgmt) $ git diff

> rbreslow@maiden raster-foundry (feature/jrb/sbtopts-mgmt) $ git add .

> rbreslow@maiden raster-foundry (feature/jrb/sbtopts-mgmt) $ git status
On branch feature/jrb/sbtopts-mgmt
Your branch is up to date with 'origin/feature/jrb/sbtopts-mgmt'.

nothing to commit, working tree clean
```
